helix-term/src/commands/typed.rs

```diff
@@ -79,12 +79,13 @@ fn buffer_close_by_ids_impl(
     doc_ids: &[DocumentId],
     force: bool,
 ) -> anyhow::Result<()> {
+    // TODO: deduplicate with ctx.block_try_flush_writes
```

Comment:
Note to self: do this

Reply:
This is actually a stale TODO, I already did this

Reply:
I copied the implementation everywhere but then removed it, forgetting the TODO: https://github.com/helix-editor/helix/pull/2267/commits/aa015cbf156af13625d73516fddddbb36060255a